### ISSUE TITLE
No Delay in HA Automations

### ISSUE THREAD

**Issue by dalethomas** [pyalarmdotcom/alarmdotcom#221]
I have been able to successfully create an automation that turns puts the alarm into Away Mode when I or my wife leaves the house.  However, when I return and enter the house there is no delay and immediately the alarm is set off.  I have confirmed this doesn't happen if I use the Alarm.com app to put the alarm in Away mode.  

Below is a screenshot of the automation in HA.  Any ideas what is causing this?  

![Screenshot 2-21-2023 at 03 27 PM](https://user-images.githubusercontent.com/7624169/220462544-42a7a611-efd7-4208-af96-bf59fb534b71.png)

**Comment by elahd** [comment]
What do your Alarm.com integration settings look like?

1. In Settings -> Devices & Services, click on Configure in the Alarm.com integration card. 
![image](https://user-images.githubusercontent.com/466460/220496858-b59fd4ea-d28d-4398-9206-837cd2343b1d.png)
2. Click "Next" to get to the Arming Mode Profiles modal. Is No Entry Delay checked for Arm Away?

![image](https://user-images.githubusercontent.com/466460/220497024-42e1b735-242a-4142-b707-65d75840fff7.png)

**Comment by RB14060** [comment]
I am having the same issue although mine apparently started over a week ago, likely when I last updated the integration. I checked my arming settings and they were all set as No Entry Delay, even though I did not change these settings. Seems like there might be a regression which is causing these to get set by default when updating.

**Comment by elahd** [comment]
Can you try again with v3.0.1?

**Comment by RB14060** [comment]
I don't believe I can test this since I am running HAOS Supervised. If there is a way to do this through the Supervised OS I'd be happy to test.

**Comment by elahd** [comment]
You can upgrade to the latest release through HACS, supervised or not.

**Comment by markus99** [comment]
Just upgraded from 2.x to 3.x a week or so ago and can confirm the 'No Entry Delay' check boxes WERE selected for each mode (Stay, Away, Night).  These were NOT selected in 2.x.  Scared the cr@p out of our dog sitter, has terrorized my wife - though fortunately we were able to prevent police from visiting.  I've unselected and saved the config.  🤞 it stays.  Love this integration - just not the immediate / unexpected alarm trigger.

**Comment by elahd** [comment]
Is this still an issue?

If so, do you run into the same problem when arming the system _manually_ through Home Assistant?

**Comment by elahd** [comment]
Closing. Please reopen if the issue persists in [v4](https://github.com/pyalarmdotcom/alarmdotcom/releases/tag/v4.0.0-beta.1).
